taxi/views.py

```diff
@@ -20,28 +25,28 @@
     return render(request, "taxi/index.html", context=context)
 
 
-class ManufacturerListView(generic.ListView):
+class ManufacturerListView(LoginRequiredMixin, generic.ListView):
```

Comment:
The `ManufacturerListView` class correctly uses `LoginRequiredMixin` to ensure that only authenticated users can access the list view.